src/main/java/io/mosip/print/service/impl/PrintRestClientServiceImpl.java

```diff
@@ -83,30 +82,25 @@ public Object getApi(ApiName apiName, List<String> pathsegments, String queryPar
 			try {
 
 				uriComponents = builder.build(false).encode();
-				printLogger.debug(uriComponents.toUri().toString(), "URI", "", "");
+				printLogger.debug(uriComponents.toUri().toString()+ "URI");
 				obj = restApiClient.getApi(uriComponents.toUri(), responseType);
 
 			} catch (Exception e) {
 				e.printStackTrace();
```

Comment:
Do not use printStackTrace anywhere in your code. Remove them in all places.

Reply:
taken care